### PR TITLE
Reformat code using PyCharm

### PR DESCRIPTION
**Pull request by AymaneHrouch**
Please review and merge so I can push new code, the code was automatically formatted using PyCharm with no intervention from my part.

### CODE REVIEW

**Comment by sudouser777** [comment]
all people may not use pycharm, it's better to use [black](https://pypi.org/project/black/) I think

**Comment by AymaneHrouch** [comment]
@sudouser777 I just wanted to format it as a start because it's really messy now. If black sounds like a good option we can use it too.
@xtekky any thoughts?

**Comment by xtekky** [comment]
it has conflics, please format the newer file

**Comment by AymaneHrouch** [comment]
@xtekky done

File: forefront/__init__.py [hunk 1]
```diff
@@ -1,51 +1,59 @@
...
 from requests import post
-from json import loads
+from tls_client import Session
+
+from forefront.mail import Mail
+from forefront.typing import ForeFrontResponse
 
 
 class Account:
+<<<<<<< HEAD
```
Comment (53jk1) [inline forefront/__init__.py:14]: Conflicts

File: forefront/__init__.py [hunk 2]
```diff
@@ -1,51 +1,59 @@
...
     @staticmethod
     def create(proxy = None, logging = False):
         
+=======
```
Comment (53jk1) [inline forefront/__init__.py:18]: Conflicts

File: forefront/__init__.py [hunk 3]
```diff
@@ -1,51 +1,59 @@
...
+    def create(proxy=None, logging=False):
+
+>>>>>>> e627601 (Reformat code using PyCharm)
```
Comment (53jk1) [inline forefront/__init__.py:21]: Conflicts

**Comment by 53jk1** [comment]
@AymaneHrouch I would like to suggest considering the implementation of a universal linter to ensure consistent code formatting across different development environments. Since different developers may have different configurations for their PyCharm IDE, having a shared linter configuration will ensure that everyone follows the same style guidelines regardless of their individual IDE settings.

**Comment by EZSNoVa** [comment]
Black is a good option, since it it available in most IDE's or text editors

**Pull request closed without merging**